iac-extensions/kubernetes/src/main/java/org/sonar/iac/kubernetes/plugin/KubernetesSensor.java

```diff
@@ -83,6 +93,13 @@ protected void initContext(SensorContext sensorContext) {
     } else {
       LOG.debug("Skipping initialization of Helm processor");
     }
+    if (sonarLintFileListener != null) {
```

Comment:
Why do we have a different analyzer than the one created from the `createAnalyzer` method?
Is the `createAnalyzer` method called at all in a context of SonarLint ?

Reply:
The different analyzer is needed for updating `ProjectContext` when files change in SonarLint.
The old `createAnalyzer` is still used by SQ & SL contexts. 
I will add a comment to document this behavior.